src/Symfony/Component/Form/Extension/Core/Type/CheckboxType.php

```diff
@@ -33,6 +34,16 @@ public function buildForm(FormBuilderInterface $builder, array $options)
         // doing so also calls setDataLocked(true).
         $builder->setData(isset($options['data']) ? $options['data'] : false);
         $builder->addViewTransformer(new BooleanToStringTransformer($options['value'], $options['false_values']));
+
+        if (!method_exists($builder, 'setIsEmptyCallback')) {
+            @trigger_error(sprintf('Not implementing the "%s::setIsEmptyCallback()" method in "%s" is deprecated since Symfony 4.4.', FormConfigBuilderInterface::class, \get_class($builder)), E_USER_DEPRECATED);
+
+            return;
+        }
+
+        $builder->setIsEmptyCallback(static function ($modelData): bool {
+            return false === $modelData;
```

Comment:
shouldn't we also to check the `null` value here?

Reply:
No IMHO, because the model data of this form type is always supposed to be a boolean. Look at the comment in the `buildForm()` method.